packages/design-system/src/helpers/notice-about-deprecation.js

```diff
@@ -0,0 +1,6 @@
+
+export const noticeAboutDeprecation = (message) => {
+    const env = process.env.NODE_ENV;
+    const isDevelopmetEnv = env === 'dev' || env === 'development'  || env === 'labs';
+    isDevelopmetEnv && console.error(message)
+}
```

Comment:
No EOL ☝️

Reply:
I would also suggest using `console.warn` instead of `console.error` for deprecation warnings.

Reply:
here some discussion about deprecation notices in React: https://github.com/facebook/react/issues/9395

Reply:
Oh, it' a good one for sure. So by using errors, there are benefits even if it is a little bit less semantically correct 🤔 Good to know 👍